eclass/mate-desktop.org.eclass

```diff
@@ -50,15 +50,11 @@ inherit versionator
 
 # Set SRC_URI or EGIT_REPO_URI based on whether live
 if [[ ${PV} == 9999 ]]; then
-	EGIT_REPO_URI="
-		https://github.com/mate-desktop/${MATE_DESKTOP_ORG_PN}.git
-		git://github.com/mate-desktop/${MATE_DESKTOP_ORG_PN}.git
-		http://github.com/mate-desktop/${MATE_DESKTOP_ORG_PN}.git
-	"
+	EGIT_REPO_URI="https://github.com/mate-desktop/${MATE_DESKTOP_ORG_PN}.git"
 	SRC_URI=""
 else
-	SRC_URI="http://pub.mate-desktop.org/releases/${MATE_BRANCH}/${MATE_DESKTOP_ORG_PN}-${MATE_DESKTOP_ORG_PV}.tar.${MATE_TARBALL_SUFFIX}"
+	SRC_URI="https://pub.mate-desktop.org/releases/${MATE_BRANCH}/${MATE_DESKTOP_ORG_PN}-${MATE_DESKTOP_ORG_PV}.tar.${MATE_TARBALL_SUFFIX}"
```

Comment:
No clue. It also resolves to 2 IPv4 addresses, so it might be two different mirrors in rotation.